build/emoji/emoji_gen.mjs

```diff
@@ -337,38 +337,40 @@ for (const key of emojiFilePositions.keys()) {
 
 const cssRules = `
 @charset "UTF-8";
+
 .emojisprite-preview {
-    // Using zoom for now as it results in less blurry emojis on Chrome - MM-34178
-    zoom: 0.55;
-    -moz-transform: scale(0.55);
+    width: ${EMOJI_SIZE_PADDED}px;
+    max-width: none;
+    height: ${EMOJI_SIZE_PADDED}px;
     background-repeat: no-repeat;
     cursor: pointer;
-    height: ${EMOJI_SIZE_PADDED}px;
-    max-width: none;
-    width: ${EMOJI_SIZE_PADDED}px;
-    padding: 0 10px 0 0;
```

Comment:
did we remove this one?

Reply:
The padding removal was intentional. I think it was compensation for some of the odd relative positioning.